environment/sdm-environment.jsonld

```diff
@@ -1,156 +1,158 @@
+		"airQualityIndex": "https://smartdatamodels.org/dataModel.Environment/airQualityIndex",
+		"airQualityLevel": "https://smartdatamodels.org/dataModel.Environment/airQualityLevel",
+		"airQualityObserved": "https://smartdatamodels.org/dataModel.Environment/airQualityObserved",
+		"airTemperature": "https://smartdatamodels.org/dataModel.Environment/airTemperature",
+		"airTemperatureTSA": "https://smartdatamodels.org/dataModel.Environment/airTemperatureTSA",
+		"alertLevel": "https://smartdatamodels.org/dataModel.Environment/alertLevel",
+		"allergenRisk": "https://smartdatamodels.org/dataModel.Environment/allergenRisk",
+		"alternateName": "https://smartdatamodels.org/alternateName",
+		"ambientNoiseTSA": "https://smartdatamodels.org/dataModel.Environment/ambientNoiseTSA",
+		"aqiMajorPollutant": "https://smartdatamodels.org/dataModel.Environment/aqiMajorPollutant",
+		"areaServed": "https://smartdatamodels.org/areaServed",
+		"arsenicTSA": "https://smartdatamodels.org/dataModel.Environment/arsenicTSA",
+		"as": "https://smartdatamodels.org/dataModel.Environment/as",
+		"atmosphericPressure": "https://smartdatamodels.org/dataModel.Environment/atmosphericPressure",
+		"atmosphericPressureTSA": "https://smartdatamodels.org/dataModel.Environment/atmosphericPressureTSA",
+		"bapTSA": "https://smartdatamodels.org/dataModel.Environment/bapTSA",
+		"benzeneTSA": "https://smartdatamodels.org/dataModel.Environment/benzeneTSA",
+		"buildingsType": "https://smartdatamodels.org/dataModel.Environment/buildingsType",
+		"c6h6": "https://smartdatamodels.org/dataModel.Environment/c6h6",
+		"cd": "https://smartdatamodels.org/dataModel.Environment/cd",
+		"cellsSize": "https://smartdatamodels.org/dataModel.Environment/cellsSize",
+		"co": "https://smartdatamodels.org/dataModel.Environment/co",
+		"co2": "https://smartdatamodels.org/dataModel.Environment/co2",
+		"co2TSA": "https://smartdatamodels.org/dataModel.Environment/co2TSA",
+		"coLevel": "https://smartdatamodels.org/dataModel.Environment/coLevel",
+		"coTSA": "https://smartdatamodels.org/dataModel.Environment/coTSA",
+		"currentLevel": "https://smartdatamodels.org/dataModel.Environment/currentLevel",
+		"dangerLevel": "https://smartdatamodels.org/dataModel.Environment/dangerLevel",
+		"dataDescriptor": "https://smartdatamodels.org/dataModel.Environment/dataDescriptor",
+		"dataProvider": "https://smartdatamodels.org/dataProvider",
+		"dateCreated": "https://smartdatamodels.org/dateCreated",
+		"dateIssued": "https://smartdatamodels.org/dataModel.Environment/dateIssued",
+		"dateModified": "https://smartdatamodels.org/dateModified",
+		"dateObserved": "https://smartdatamodels.org/dateObserved",
+		"dateObservedFrom": "https://smartdatamodels.org/dataModel.Environment/dateObservedFrom",
+		"dateObservedTo": "https://smartdatamodels.org/dataModel.Environment/dateObservedTo",
+		"depth": "https://smartdatamodels.org/dataModel.Environment/depth",
```

Comment:
this file is basically a copy of the Environment data model from SDM and as such is already a bit crappy. I would suggest not adding things in it (it is anyway meant to be removed in the cherry-picking operation)